src/Shared/Recruit.Vacancies.Client/Domain/Repositories/IVacancyReviewQuery.cs

```diff
@@ -9,7 +9,7 @@ public interface IVacancyReviewQuery
     {
         Task<List<T>> GetActiveAsync<T>();
         Task<List<VacancyReview>> GetForVacancyAsync(long vacancyReference);
-        Task<List<VacancyReview>> SearchAsync(long vacancyReference);
+        Task<VacancyReview> SearchByReferenceAsync(long vacancyReference);
```

Comment:
There can be many reviews for a single VacancyReference so should this be renamed to describe the review it is retrieving? `GetLatestReviewByReference`? Or something..